pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/MetsData.scala

```diff
@@ -0,0 +1,47 @@
+package uk.ac.wellcome.platform.transformer.mets
+
+import uk.ac.wellcome.models.work.internal._
+
+case class MetsData(
+  recordIdentifier: String,
+  accessCondition: Option[String]
+) {
+
+  def toWork(version: Int) =
+    // TODO: change this to UnidentifiedInvisibleWork (UnidentifiedPartialWork?)
+    UnidentifiedWork(
+      title = "??? SHOULD TITLE BE OPTIONAL ???",
```

Comment:
Hmm, good point... I think there might be places that rely on this not being optional, probably where we serialize into display models. So making this optional means deciding what should the API do if it finds a work with no title